module2/collections.py

```diff
@@ -0,0 +1,38 @@
+import random
+import string
+
+# Part 1
+# Create list if random number of dicts.
+random_dict = dict()
+random_list = []
+for i in range(random.randint(2, 10)):
+    for j in range(random.randint(0, 10)):
+        random_dict[random.choice(string.ascii_lowercase)] = random.randint(0, 1000)
+    random_list.append(random_dict.copy())
```

Comment:
why rewrite random_dict as {} after each iteration?

Reply:
random_dict.clear() added for generating 'more random' dicts, because  if we don't clean previous dict and don't change value for key in current iteration that this key:value pair will be added to 'next' dict.

Reply:
I mean you can move line 6 between 8 and 9 and nothing will change :) Not really mistake, just python-specific way to iterate. And I think you don't really need to copy random dict, it will be done automatically after you rewrite random_dict variable. Actually none of them are mistakes. But if you want to taste cool python features, try to replace all these lines 6-12 with one-line expression using list 
 and dict comprehensions :)

Reply:
tried to replace :)